src/lib/content.js

```diff
@@ -118,6 +119,14 @@ function parseMarkdownContent(data) {
 	});
 }
 
+function addFragmentLinksToHeadings(data) {
+	data.html = data.html.replaceAll(
+		/(<h[1-6] id="([^"]+)">)([^\n]+)/g,
+		'$1<a href="#$2"></a>$3'
+	);
+	return data;
+}
```

Comment:
If we upgrade `marked` we can hook directly in their markdown renderer instead of regexing the HTML after all is done.

Reply:
Oh neat, didn't realize it supported that. Will take a look tomorrow

Reply:
Oh man, 9 major versions. Will need time to verify everything's working correctly

Reply:
So `marked` has tore through 5 majors in the past 4 months, including ripping some things we use. I don't think it makes sense to touch that now, especially as the docs don't seem to be in a good state.